Modules/_tkinter.c

```diff
@@ -321,8 +321,6 @@ static PyObject *Tkinter_TclError;
 static int quitMainLoop = 0;
 static int errorInCmd = 0;
 static PyObject *excInCmd;
-static PyObject *valInCmd;
```

Comment:
These were unused before this PR?

Reply:
They were used in a couple of Restore() calls, scroll down.